src/backend/config/configuration.template.yaml

```diff
@@ -5,8 +5,6 @@ deployments:
     - sagemaker
     - azure
     - bedrock
-  cohere_platform:
```

Comment:
Why are we removing this?

Reply:
It's duplicated - there's the same setting in the secrets template and I figured it's more appropriate there